ci/bullseye_generate_report_node.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -ex
+
+sudo mkdir -p "$DAOS_BASE"
+sudo mount -t nfs "$HOSTNAME":"$HOSTPWD" "$DAOS_BASE"
+
+set +x
```

Comment:
Why is this needed?

Reply:
Needs to have the workspace mounted in the test VM where the web site is created.   In the future if the stage is refactored to run in a docker container it will not be needed.

Reply:
I mean specifically the disabling of _xtrace_ (`set +x`).

Reply:
Need to suppress echoing of the BULLSEYE environment variable, even though Jenkins should mask it.
I am reworking this code today to use a dockerfile, which should simplify it a bit.